sensor/common/compliance/nodescan_handler_test.go

```diff
@@ -20,7 +18,11 @@ func assertNoGoroutineLeaks(t *testing.T) {
 	)
 }
 func TestNodeScanHandler(t *testing.T) {
```

Comment:
Nitpicks.
1. I wonder how does it pass linters without a blank line between these two functions?
```suggestion
}

func TestNodeScanHandler(t *testing.T) {
```
2. I'd suggest to put `assertNoGoroutineLeaks` closer to where it's called.